interface/patient_tracker/patient_tracker.php

```diff
@@ -629,9 +629,15 @@ class="text-small"
                         <?php if ($GLOBALS['ptkr_show_encounter']) { ?>
                             <td class="detail text-center" name="kiosk_hide">
                                 <?php
+                                $signed = false;
                                 if ($appt_enc != 0) {
+                                    $signed = is_signed($appt_enc);
```

Comment:
why not use the `isSigned()` method of the the esign api like is done in the [custom report](https://github.com/openemr/openemr/blob/master/interface/patient_file/report/custom_report.php#L805)?

Reply:
On the patient tracker, the information that is available is the encounter number only. No form information is give in the current data set of the patient tracker. The API requires a for name that is not available at run time.

Reply:
Hi @stephenwaite, AI let me open the black-box and see what was inside. It found for me what I had forgotten was in the black-box ESign

Reply:
I tested the change in production, and it is producing the right results.